src/Sarif.Driver.UnitTests/Sdk/FormatForVisualStudioTests.cs

```diff
@@ -177,14 +177,11 @@ private Result MakeResultFromTestCase(ResultKind kind, Region region)
                 {
                     new Location
                     {
-                         AnalysisTarget = new List<PhysicalLocationComponent>
-                         {
-                             new PhysicalLocationComponent
-                             {
-                                 Uri = TestAnalysisTarget.CreateUriForJsonSerialization(),
-                                 Region = region
-                             },
-                         }
+                        AnalysisTarget = new PhysicalLocation
+                        {
+                            Uri = TestAnalysisTarget.CreateUriForJsonSerialization(),
+                            Region = region
+                        }
```

Comment:
Every place you see a change like this, it's because `PhysicalLocation` is now its own object, not an array of `PhysicalLocationComponent` objects. The `Uri` is the key into the `FileInfo` dictionary.